app/views/document_associations/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_for :back_link, document_path(@document) %>
+<% content_for :title, "Document associations" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_tag(document_associations_path(@document)) do %>
+
+      <% @document.document_type_schema.associations.each do |field| %>
+        <%= render "document_associations/fields/#{field.type}_input", name: field.id, label: field.label, associations: @document["associations"] %>
```

Comment:
Should we pass in `associations: @document["associations"][name]` here instead?

Reply:
yeah that'd probably be better. I got caught in a halfway house between how it's done elsewhere.